tekton/src/pipelines/fvt-sls.yml.j2

```diff
@@ -32,3 +32,15 @@ spec:
     # 1. SLS FVT
     # -------------------------------------------------------------------------
     {{ lookup('template', 'taskdefs/fvt-sls/sls.yml.j2') | indent(4) }}
+    - name: imagescan
```

Comment:
I don't think we can add this for sls yet .. it's not a real MAS app, would leave it out for now ... get the apps done first.  this will need special handling